### ISSUE TITLE
Don't allow or warn against editing if location is 0,0

### ISSUE THREAD

**Issue by SomeoneElseOSM** [bryceco/GoMap#514]
Hello,

There have been a couple of examples recently (most recently https://www.openstreetmap.org/changeset/102672041 ) where someone's started editing before their device knows where it is.  This can result in things created around latitude 0, longitude 0 ("null island"), or things there being nudged slightly.  There is a weather monitoring buoy there https://www.openstreetmap.org/node/3815077900 that takes most of the flak, but buildings do also get occasionally created there too.  Most often the culprit is a calculation cockup prior to importing some data with JOSM, but there have been a couple of recent examples of Go Map! edits there too.

Whether this is a Go Map! thing or actually a recent iOS change I've no idea of course.

Regards,
Andy

**Comment by bryceco** [comment]
Yeah that's a good idea to simply disallow creation or changes to objects in that vicinity. Thanks for the report!
